src/java.desktop/share/classes/javax/swing/plaf/basic/BasicTextUI.java

```diff
@@ -303,8 +303,8 @@ private void updateBackground(JTextComponent c) {
     /**
      * Initializes component properties, such as font, foreground,
      * background, caret color, selection color, selected text color,
-     * disabled text color, and border color.  All properties are set
-     * if their current value is either null or a UIResource.
+     * disabled text color, border and margin. Each property is set
```

Comment:
```suggestion
     * disabled text color, border, and margin. Each property is set
```
I suggest keeping the command before “and” as it was before.

Reply:
I think these are nouns and as per I know, we shouldn't place comma before "and" when we are simply listing the items.

Reply:
> I think these are nouns and as per I know, we shouldn't place comma before "and" when we are simply listing the items.

This is called [Oxford Comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/). It is quite standard in American English and javadoc uses American English. The comma was present before therefore I suggest following the style and add the now missing comma.

Reply:
ok